src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -50,6 +50,7 @@ public function collect(Request $request, Response $response, \Exception $except
                 'enabled' => false,
                 'authenticated' => false,
                 'token_class' => null,
+                'provider_key' => null,
```

Comment:
I agree. However, the method in the token is called `getProviderKey()`. That's why I prefer to be consistent here.